client/components/results-list.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-export default function ResultsList({ searchResult, searchTerm }) {
+export default function ResultsList({ searchResult, searchTerm, handleAnchorClick, images, carouselImages, currentIndex }) {
```

Comment:
My preference is to name the callback functions starting with `on` so they look like DOM event handlers. So, `onAnchorClick` for example.

This is just a preference, but most of the React code I work with follows the same convention.